hooks/useNetworkStatus.ts

```
@@ -0,0 +1,131 @@
+        return 'Cellular';
+      case Network.NetworkStateType.ETHERNET:
+        return 'Ethernet';
+      case Network.NetworkStateType.BLUETOOTH:
+        return 'Bluetooth';
+      case Network.NetworkStateType.VPN:
+        return 'VPN';
+      case Network.NetworkStateType.WIMAX:
+        return 'WiMAX';
+      case Network.NetworkStateType.NONE:
+        return 'None';
+      case Network.NetworkStateType.UNKNOWN:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)